CHANGELOG.md

```diff
@@ -21,6 +24,9 @@
 
 ### Changed
 
+- Bump kedro requirement to <=0.16.5
+- `kedro mlflow init` is no longer touching the run.py file
```

Comment:
We should likely add a test for the kedro version to ensure retro compatibility, or make it very clear in the docs that if you have kedro ">=0.16,<0.16.4".

Reply:
yes, we should discuss this particular point in a separate PR.